packages/actor-init-sparql/lib/ActorInitSparql-browser.ts

```diff
@@ -236,3 +239,4 @@ export interface IActorInitSparqlArgs extends IActorArgs<IActionInit, IActorTest
 export const KEY_CONTEXT_INITIALBINDINGS: string = '@comunica/actor-init-sparql:initialBindings';
 export const KEY_CONTEXT_QUERYFORMAT: string = '@comunica/actor-init-sparql:queryFormat';
 export const KEY_CONTEXT_BASEIRI: string = '@comunica/actor-init-sparql:baseIRI';
+export const KEY_CONTEXT_QUERY_TIMESTAMP: string = '@comunica/actor-init-sparql:queryTimestamp';
```

Comment:
Could you also add a context shortcut here?: https://github.com/comunica/comunica/blob/master/packages/actor-init-sparql/components/Actor/Init/Sparql.jsonld#L101-L109